Frontend/business-expense-manager/src/services/category.service.ts

```diff
@@ -20,4 +21,10 @@ export class CategoryService {
       .get<ApiResponse<Category[]>>(`${this.baseUrl}/category`);
   }
 
+  addCategoryBudget(categoryBudget: CreateCategoryBudgetDto) : Observable<ApiResponse<number>> {
+    return this
+      .httpClient
+      .post<ApiResponse<CreateCategoryBudgetDto>>(`${this.baseUrl}/monetary_flow/add`, categoryBudget)
```

Comment:
I think the url is wrong, but that's fine. We will fix when we integrate the services.